app/code/community/Lesti/Fpc/Model/Observer.php

```diff
@@ -212,10 +212,13 @@ protected function _prepareLayout(
         array $dynamicBlocks
     )
     {
-        $xml = simplexml_load_string(
-            $layout->getXmlString(),
-            Lesti_Fpc_Helper_Data::LAYOUT_ELEMENT_CLASS
-        );
+        $xml = $layout->getNode();
+        if (!is_a($xml, Lesti_Fpc_Helper_Data::LAYOUT_ELEMENT_CLASS)) {
```

Comment:
Maybe this `if` is also superfluous, cause all we do with `$xml` is just calling `xpath` function on it and that is a [SimpleXMLElement](https://secure.php.net/manual/en/simplexmlelement.xpath.php) function. As [getNode](https://github.com/OpenMage/magento-mirror/blob/1.9.3.2/lib/Varien/Simplexml/Config.php#L146) is returning a [Varien_Simplexml_Element](https://github.com/OpenMage/magento-mirror/blob/1.9.3.2/lib/Varien/Simplexml/Element.php#L35) that is extending from [SimpleXMLElement](https://secure.php.net/manual/en/class.simplexmlelement.php), we should be safe without the `if` or?